package.json

```
@@ -3,6 +3,25 @@
   "version": "1.0.0",
   "description": "Cross-border remittance platform for Zimbabwean expatriates in South Africa",
   "private": true,
+  "author": "PacheduConnect Development Team",
```

Biome (2.1.2)

[error] The key author was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)